rust/datafusion/src/utils.rs

```diff
@@ -74,6 +74,29 @@ macro_rules! make_string {
     }};
 }
 
+macro_rules! make_string_from_list {
+    ($column: ident, $row: ident) => {{
+        let list = $column
+            .as_any()
+            .downcast_ref::<array::ListArray>()
+            .unwrap()
```

Comment:
`expect` would be better than `unwrap`. Using a `Result` would be better than `expect`.